### PR TITLE
Adding missing german translation file for validations

### PR DESCRIPTION
**Pull request by andesk**

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@andesk thanks a lot for providing this translation. I'm sorry to bother you, but today we added 1 new string to translate:

```xml
 <trans-unit id="post.blank_summary">
    <source>post.blank_summary</source>
    <target>Gib deinem Beitrag eine Zusammenfassung!</target>
</trans-unit>
<trans-unit id="post.blank_content">
    <source>post.blank_content</source>
    <target> ... NEW TRANSLATION HERE ... </target>
</trans-unit>
```

**Comment by xabbuh** [comment]
@javiereguiluz Can you point to the commit where these strings have been added? It seems like all other translations do miss those translations do or they are named differently.

**Comment by javiereguiluz** [comment]
@xabbuh here: https://github.com/symfony/symfony-demo/commit/2c9a153e2da8378e438bb42510ebc552f5e743a9

**Comment by xabbuh** [comment]
@javiereguiluz Oh indeed (I was looking in `messages.en.xliff`).

File: app/Resources/translations/validators.de.xliff [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="de" datatype="plaintext" original="file.ext">
+        <body>
+        <trans-unit id="post.blank_summary">
+            <source>post.blank_summary</source>
+            <target>Gib deinem Beitrag eine Zusammenfassung!</target>
+        </trans-unit>
+        <trans-unit id="post.too_short_content">
+            <source>post.too_short_content</source>
+            <target>Der Beitragsinhalt ist zu kurz (mindestens {{ limit }} Zeichen)</target>
```
Comment (xabbuh) [inline app/Resources/translations/validators.de.xliff]: Missing closing exclamation mark (same below).

Comment (andesk) [inline app/Resources/translations/validators.de.xliff]: There are no exclamation marks in the english/french/... translation file as well.

**Comment by andesk** [comment]
German translation for blank_content was added.

**Comment by javiereguiluz** [comment]
Thank you @andesk

**Pull request closed without merging**